content/doc/book/blueocean/activity.adoc

```diff
@@ -19,93 +19,72 @@ endif::[]
-Clicking the "Favorite" symbol (a star outline "&#9734;") adds a branch to the favorites list shown on the
-<<dashboard#favorites, Dashboard's "Favorites" list>> for this user.
-A favorite branch will show a solid star "&#9733;" and clicking it removes
-this branch from the favorites.
-
-== Pull Requests
-
-The "Pull Requests" tab shows a list of all
-Pull Requests for the current Pipeline that have a completed or in-progress Run.
-(Some source control systems call these "Merge Requests", others do not support them at all.)
-Each line in the list corresponds to a pull request in source control, showing
-the status of the most recent Run: id number, commit information, duration, and when the Run completed.
+* *Pipeline Name* - Selecting this displays the <<activity#, default Activity tab>>
+* *Favorites Toggle* - Selecting the favorite icon &#9734; to the right of the Pipeline name, adds a branch to the favorites list shown on the <<dashboard#favorites, dashboard's favorites list>>.
+* *Tabs* (<<activity, Activity>>, <<branches, Branches>> <<pull-requests, Pull Requests>>) - Selecting one of these will navigate to the corresponding information in the Activity view.
+
+=== Activity
+
+The default tab of the Activity view shows a list of the latest completed or in-progress runs.
+Each line represents the <<dashboard#run-status, status>> of a run, run id, commit information, and when the run completed.
+
+image:blueocean/activity/overview.png[Default activity tab view of the Activity view, role=center]
+
+* Selecting a run will bring up the <<pipeline-run-details#, Pipeline run details>> to provide Pipeline visualization.
+* In-progress runs can be aborted from this list by selecting the *stop* icon, which is represented by a &#9726; within a circle.
+* Runs that have been completed can be re-run by selecting the
+*re-run* icon &#8634;.
+* The list can be filtered by branch using the "Branch" drop-down in the list header.
+image:blueocean/activity/branch-filter.png[Branch filter in Blue Ocean activity view, width=17% role=center]
+
+This view does not allow runs to be edited or marked as favorites.
+To perform these actions, select the "<<branches, Branches>>" tab.
+
+=== Branches
+
+The Branches tab shows a list of all branches that have a completed or in-progress run in the current Pipeline.
+Each line in the list corresponds to a branch in source control, showing <<dashboard#pipeline-health, overall health of the branch>> based on the status of the most recent run, branch name, commit information, and when the run completed.
+
+image:blueocean/activity/branches.png[Branches tab of Activity view, role=center]
+
+Selecting a branch will bring up the <<pipeline-run-details#, Pipeline run details>> for the latest completed or in-progress run of that branch.
+
+. Pipelines where the latest run has been completed can be run again by selecting the *run* icon, represented by a icon:play[] in a circle.
```

Comment:
Small suggestion, take it or leave it. :) 
```suggestion
. Pipelines where the latest run has been completed can be rerun by selecting the *run* icon, represented by a icon:play[] in a circle.
```

Reply:
Re-run is a separate icon in this case, so I wanted to try and be a bit more specific. However, I can see where that may not be clear enough in this case and will update the wording.